vpr/src/pack/greedy_candidate_selector.h

```diff
@@ -516,5 +517,9 @@ class GreedyCandidateSelector {
     /// @brief A count on the number of unrelated clustering attempts which
     ///        have been performed.
     int num_unrelated_clustering_attempts_ = 0;
+
+    /// @brief Random number generator to get a random atom between 0 and
```

Comment:
I'd make this slightly more general to start.
I'd change this to:
@brief Random number generator used by the clusterer. Currently this is used only when selecting atoms from
attraction groups, but could be used for other purposes in the future.